scripts/tags/exturl.js

```diff
@@ -4,8 +4,11 @@
 
 'use strict';
 
-var util = require('hexo-util');
+/*jshint camelcase: false */
+var util = require(hexo.base_dir + 'node_modules/hexo/node_modules/hexo-util');
```

Comment:
Please take a look at [npm v3 Dependency Resolution](https://docs.npmjs.com/how-npm-works/npm3). For this case, `hexo-util` is a dependency of Hexo, but is will be installed in `node_module` just alone with `hexo` module (not in the sub-directory of `hexo` module). 

I tested it with latest stable Node.js and NPM and [the issue reporter](https://github.com/iissnan/hexo-theme-next/issues/1490#issuecomment-286452797) has tested it as well.

So the question is that users who use npm lower than version 3 will encounter the error `hexo-util` module not found.